store/mongo/migration_1_1_1.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2024 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package mongo
+
+import (
+	"context"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	mopts "go.mongodb.org/mongo-driver/mongo/options"
+
+	"github.com/mendersoftware/go-lib-micro/mongo/migrate"
+)
+
+const (
+	IndexNameTTL = `log_TTL`
+)
+
+type migration_1_1_1 struct {
+	client *mongo.Client
+	db     string
+}
+
+// Up creates indexes for fetching event documents
+func (m *migration_1_1_1) Up(from migrate.Version) error {
+	ctx := context.Background()
+	eventModel := mongo.IndexModel{
+		Keys: bson.D{
+			{Key: KeyEventTs, Value: 1},
+		},
+		Options: mopts.Index().
+			SetName(IndexNameTTL).
+			SetExpireAfterSeconds(0),
```

Comment:
maybe I am lost, so where do we set after how long the record is removed?

Reply:
It is [set on insertion](https://github.com/mendersoftware/iot-manager/blob/41a7b3526c4e15d8af226105abf12e7ae59e905a/store/mongo/datastore_mongo_events.go#L84).

Reply:
Oops that made me realize, I'm indexing the wrong key! :anguished:
Fixing...